packages/sit-onyx/src/components/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+import type { OnyxNavItemProps } from "./types";
+import OnyxListbox from "../OnyxListbox/OnyxListbox.vue";
+import type { ListboxOption } from "../OnyxListbox/types";
+
+const props = withDefaults(defineProps<OnyxNavItemProps>(), {
+  active: false,
+});
+
+const emit = defineEmits<{
+  /**
+   * Emitted when the nav item is clicked (via click or keyboard).
+   */
+  navigate: [href: string];
+}>();
+
+defineSlots<{
+  /**
+   * An optional slot to show additional content behind the label (e.g. a `OnyxBadge`).
+   */
+  default?(): unknown;
+}>();
+
+const listboxOptions = computed<ListboxOption<string>[]>(() => {
+  return props.options?.map((opt) => ({ id: opt.href, label: opt.label })) ?? [];
+});
+</script>
+
+<template>
+  <li
+    role="menuitem"
+    tabindex="0"
+    :aria-label="props.label"
+    class="onyx-nav-item onyx-text"
+    :class="{ 'onyx-nav-item--active': props.active || props.options?.find((opt) => opt.active) }"
+    @click="props.href && emit('navigate', props.href)"
+    @keydown.enter="props.href && emit('navigate', props.href)"
+  >
+    <span>{{ props.label }}</span>
+    <slot></slot>
+  </li>
+  <OnyxListbox
+    v-if="listboxOptions.length > 0"
+    class="onyx-nav-item-listbox"
+    :label="'Options of ' + props.label"
```

Comment:
Hmm, no harm in translating it. I should make it a property right?

Reply:
I thinks its enough to just add a translation because the translation can already be overriden by the projects (which is propably not needed in most cases)